lib/types/src/compilation/target.rs

```diff
@@ -192,6 +192,13 @@ impl Target {
     pub fn cpu_features(&self) -> &EnumSet<CpuFeature> {
         &self.cpu_features
     }
+
+    /// Check if target is a native (eq to host) or not
+    pub fn is_native(&self) -> bool {
+        let host = Triple::host();
+        host.operating_system == self.triple.operating_system
+            && host.architecture == self.triple.architecture
```

Comment:
I believe you can do simple compare on the triple itself.

```suggestion
host == self.triple
```

Reply:
We take the risk of comparing Vendor for example, that should could lead to a different result while it should not.
I did this to be sure to have a minimum comparison.